src/main/c/Jep/pyembed.c

```diff
@@ -356,9 +358,15 @@ void pyembed_startup(JNIEnv *env,
 
     PyStatus status = PyStatus_Ok();
     PyConfig config;
-    PyConfig_InitPythonConfig(&config);
+    if (isolated) {
+        PyConfig_InitIsolatedConfig(&config);
+    } else {
+        PyConfig_InitPythonConfig(&config);
+    }
     // According to PEP-587 the fields shared with PyPreConfig should be set first.
-    config.parse_argv = 0;
+    if (parseArgv >= 0) {
+        config.parse_argv = 0;
+    }
```

Comment:
This confuses me.  The options are 0 or 1, so how do you set it to 1 with this code?    https://docs.python.org/3/c-api/init_config.html#c.PyConfig.parse_argv

Reply:
That is definitely an oversight on my part. Thanks for pointing it out. I've fixed it in the latest commit.